addons/repair/i18n/repair.pot

```diff
@@ -729,6 +729,14 @@ msgstr ""
 msgid "Number of errors"
 msgstr ""
 
+#. module: repair
+#. odoo-python
+#: code:addons/repair/models/repair.py:0
+#, python-format
+msgid ""
+"Note that the warehouse of the return and repair locations doesn't match!"
+msgstr ""
```

Comment:
Is this really the right place in the pot file ? we order the translated strings alphabetically and I doubt it goes there automatically when you export the translations :thinking: 

(same remark for the diff in `stock.pot`)

Reply:
@Feyensv thanks and updated the translation files